### PR TITLE
Screen 6 - Updates and other software

### PR DESCRIPTION
**Pull request by charafau**
Implemented screen 6 design with model 

image

issue: https://github.com/canonical/ubuntu-desktop-installer/issues/31

### CODE REVIEW

File: packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  Widget build(BuildContext context) {
+    final model = context.watch<UpdateOtherSoftwareModel>();
+    return LocalizedView(
+      builder: (context, lang) => Scaffold(
+        appBar: AppBar(
+          title: Text(lang.updatesOtherSoftwarePageTitle),
+          automaticallyImplyLeading: false,
+        ),
+        body: Container(
+          padding: const EdgeInsets.all(20),
```
Comment (jpnurmi) [inline packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart:35]: could the padding be around the description label and as content padding of the radio list tiles instead?

image

File: packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                onChanged: model.setInstallationMode,
+              ),
+              RadioListTile<InstallationMode>(
+                title: Text(lang.minimalInstallationTitle),
+                subtitle: Text(lang.minimalInstallationSubtitle),
+                value: InstallationMode.minimal,
+                groupValue: model.installationMode,
+                onChanged: model.setInstallationMode,
+              ),
+              SizedBox(height: 24),
```
Comment (jpnurmi) [inline packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart:57]: nit-pick: could be const :)

**Review by jpnurmi — COMMENTED** [review]
Nice and tidy :+1:

File: packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+              Align(
+                alignment: Alignment.centerLeft,
+                child: Text(lang.otherOptions),
+              ),
+              CheckboxListTile(
+                  title: Text(lang.installThirdPartyTitle),
+                  subtitle: Text(lang.installThirdPartySubtitle),
+                  controlAffinity: ListTileControlAffinity.leading,
+                  value: model.installThirdParty,
+                  onChanged: model.setInstallThirdParty)
```
[the remark this answers is not in the document]
Comment (charafau) [reply packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart]: done

File: packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart (hunk 1 above)
Comment (charafau) [reply packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart]: done

File: packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart (hunk 2 above)
Comment (charafau) [reply packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart]: done

**Comment by jpnurmi** [comment]
LGTM :+1:

**Comment by fcore117** [comment]
Does this new installer finally install minimal really minimal?
https://github.com/canonical/ubuntu-desktop-installer/issues/31#issuecomment-803913309

**Pull request merged**